packages/wdio-browserstack-service/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "typeScriptVersion": "3.8.3",
   "dependencies": {
+    "@browserstack/ai-sdk-node": "git+ssh://git@github.com/browserstack/ai-sdk-node.git#431416dcdf8a01644fb2e836d34b48d3dcf2034b",
```

Comment:
Replace with correct version.

Reply:
This can be done once a working version of AI-SDK is published to npm